### PR TITLE
#6, #7 Datamodel

### LINKED ISSUE #6: Create relationship between users (of the app) and persons (of interest)
2 PUNKTE

- Entsprechend dem entstandenen Konzept (#5)

### LINKED ISSUE #7: Create database according to the concept
3 PUNKTE

Entsprechend der Konzeptionierung (#5) die Datenbank im Projekt erstellen

### PR DESCRIPTION
**Pull request by lwenner**
**- Summary**
This PR implements the data model as described in #7 . In addition, the User Person relation (#6) was also implemented.

Closes #7 , Closes #6

### CODE REVIEW

**Review by cdfhalle — DISMISSED** [review]
Die Fehler die der Linter aktuell wirft, sollten durch #54 gefixt werden

File: app/models/chair.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Chair < ApplicationRecord
+  validates :name, presence: true
```
Comment (Zyranix) [inline app/models/chair.rb:2]: In the model, we have a 1..*-relation to Person. Should we validate if there exists at least one person?

File: app/models/room.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Room < ApplicationRecord
+  validates :full_name, presence: true
+
+  has_and_belongs_to_many :room_types
```
Comment (Zyranix) [inline app/models/room.rb:4]: Same as above: 1...* relation to Room Type

File: app/models/chair.rb (hunk 1 above)
Comment (Zyranix) [reply app/models/chair.rb:2]: Maybe something like `validates :person, presence: true` works?

**Review by Zyranix — DISMISSED** [review]
Looks good 👍🏻 
Fully implements and validates the proposed database scheme in #5

**Review by cdfhalle — DISMISSED** [review]

**Review by Zyranix — DISMISSED** [review]

**Review by Zyranix — APPROVED** [review]

**Pull request merged**